doc/changelog.md

```diff
@@ -1,55 +1,100 @@
-# Release 0.3.2-dev
+# Release 0.3.2
 
 <h3>New features</h3>
 
-<h3>Improvements</h3>
+* The experimental AutoGraph feature now supports Python `while` loops, allowing native Python loops
+  to be captured and compiled with Catalyst.
+  [(#318)](https://github.com/PennyLaneAI/catalyst/pull/318)
 
-* Improve the compiler driver diagnostic output even more. Now it dumps the failing IR as well as
-  prints names of failing passes.
-  [(#349)](https://github.com/PennyLaneAI/catalyst/pull/349)
+  ```python
+  dev = qml.device("lightning.qubit", wires=4)
```

Comment:
I moved this to a separate call, as there is still the issue that memory allocation happens on device instantiation, and until this is fixed, want to avoid repeating device instantiation in the QNode decorator becoming a user pattern